osu.Game/Beatmaps/IWorkingBeatmap.cs

```diff
@@ -17,33 +17,69 @@ namespace osu.Game.Beatmaps
 {
     public interface IWorkingBeatmap
     {
+        IBeatmapInfo BeatmapInfo { get; }
+
+        IBeatmapSetInfo BeatmapSetInfo => BeatmapInfo.BeatmapSet;
+
+        IBeatmapMetadataInfo Metadata => BeatmapInfo.Metadata;
+
+        /// <summary>
+        /// Whether the Beatmap has finished loading.
+        ///</summary>
+        public bool BeatmapLoaded { get; }
```

Comment:
How do these `public`s sometimes appear and disappear? I don't mention them anymore because it's a lost cause, but is this some sort of automatic refactor doing it?

Reply:
It's manual - I copy the methods across from non-interface to interface and forget to remove the `public` prefix.